src/aws/bin/terraform

```diff
@@ -8,6 +8,6 @@ docker run --rm -it \
     -u "$(id -u)" \
     -v "${PWD}/:/app" \
     -w "/app" \
-    --env-file ./env.d/development \
+    --env-file ./env.d/${MARSHA_TERRAFORM_ENV_FILE:-development} \
```

Comment:
Just for information, I suppose MARSHA_TERRAFORM_ENV_FILE will be set directly in the CI ?

Reply:
Terraform is not deploy on the CI. It will be used on a local machine.

Reply:
Ok, then I suppose there's a need to use other files on local machine.
But, where should we define MARSHA_TERRAFORM_ENV_FILE variable ?

Reply:
Yes I have several files on my machine. 
You will define it at runtime
```
$ MARSHA_TERRAFORM_ENV_FILE=production ./bin/terraform apply`

Reply:
Thanks for your explanations :)

Reply:
@lunika why not pass directly the variable that we want to set instead of modifying a file and then passing the file!?

Reply:
Do you have an example ? I don't understand what you mean.